src/code.ts

```diff
@@ -52,7 +52,7 @@ export class Code {
 
   inspect(): string {
     const codeJson = this.toJSON();
-    return `new Code("${codeJson.code}"${
+    return `new Code("${codeJson.code.toString()}"${
```

Comment:
Do you want to do

```suggestion
    return `new Code("${String(codeJson.code)}"${
```

here as well, since there is no argument validation in the constructor and this handles null/undefined as well?